docs/pql-guide/pql-home.md

```diff
@@ -61,3 +61,63 @@ PQL queries can be broadly grouped into:
 | Read operations | Comprise the majority of PQL keywords, and often the majority of PQL usage. | [PQL Read operations](/docs/pql-guide/pql-read-home) |
 | Write operations | Write operations can be used instead of command-line ingestion tools | [PQL Write operations](/docs/pql-guide/pql-write-home)<br/>[CLI ingest tools](/docs/community/com-ingest/com-ingest-manage) |
 | Other operations | Modify the execution context and return types of the queries passed to it. | [PQL options](/docs/pql-guide/pql-options) |
+
+
+<!--
```

Comment:
not PQL, all com ingest related and should not be here

Reply:
Removing now